### PR TITLE
Feat/volumes

### PR DESCRIPTION
**Pull request by pbarbarant**
Corrected PR strictly for the addition of volumetric features. Examples are now written in  ``.py`` instead of ``.ipynb``.

### CODE REVIEW

File: examples/02_volumetric_alignment/plot_1_aligning_low_res_volumes.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    target_geometry=target_geometry_normalized,
+    solver="sinkhorn",
+    solver_params={
+        "nits_bcd": 3,
+    },
+    verbose=True,
+)
+
+# %%
+# Let's plot the probability map of the target voxels being matched with
```
Comment (alexisthual) [inline examples/02_volumetric_alignment/plot_1_aligning_low_res_volumes.py:147]:
```suggestion
# Let's plot the probability map of target voxels being matched with
```

File: examples/02_volumetric_alignment/plot_1_aligning_low_res_volumes.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+ax.set_title(
+    "Probability map of target vertices\n"
+    f"being matched with source point {vertex_index}"
+)
+ax.set_axis_off()
+fig.colorbar(plt.cm.ScalarMappable(cmap="twilight"), ax=ax)
+plt.show()
+
+# %%
+# We can now align the test contrasts using the fitted mapping.
```
Comment (alexisthual) [inline examples/02_volumetric_alignment/plot_1_aligning_low_res_volumes.py:183]:
```suggestion
# We can now align test contrasts using the fitted mapping.
```

**Review by alexisthual — COMMENTED** [review]
Almost there! :muscle: 

Our CI is currently complaining because older examples make use of `compute_lmds()`, which was renamed to `compute_lmds_mesh()`.

I have made a few suggestions, mostly typos and naming.

Here is a list of ideas to track for future reference:
- adding the `method` parameter to `compute_lmds_mesh()`
- checking what happens when our volumetric segmentation has more than one connected component

File: src/fugw/scripts/lmds.py [hunk 3]
```diff
@@ -195,3 +175,149 @@ def compute_lmds(
...
+    basis_distance = basis_distance.type(torch.float32)
+
+    return _compute_embedding(
+        basis_distance, n_landmarks, n_voxels, k, tol, invert_indices
+    )
+
+
+def compute_lmds_volume(
+    segmentation,
+    method="geodesic",
```
[the remark this answers is not in the document]
Comment (pbarbarant) [reply src/fugw/scripts/lmds.py:245]: Good idea !

File: src/fugw/scripts/lmds.py [hunk 4]
```diff
@@ -195,3 +175,149 @@ def compute_lmds(
     X = X[invert_indices]
 
     return X
+
+
+def compute_lmds_mesh(
+    coordinates,
+    triangles,
```
[the remark this answers is not in the document]
Comment (pbarbarant) [reply src/fugw/scripts/lmds.py:182]: Okay, let's keep that in mind.

**Pull request merged**